examples/seq2seq/utils.py

```diff
@@ -42,11 +42,11 @@ def encode_file(
     examples = []
     for text in tqdm(lns, desc=f"Tokenizing {data_path.name}"):
         tokenized = tokenizer(
-            [" " + text],
+            [text],
```

Comment:
Is that left-over code from the T5 preprocessing?

Reply:
no it was a hack to replicate add_prefix_space=True. Does `T5Tokenizer` need that kwarg?